src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
I think we should have an app context. It allows you to drill down to sub contexts when you need to. For example, in GitLink there was no context in the beginning. Then what is holding the final argument options / information?